src/cmap/connection.ts

```diff
@@ -526,7 +526,11 @@ export class Connection extends TypedEventEmitter<ConnectionEvents> {
             new CommandSucceededEvent(
               this,
               message,
-              options.noResponse ? undefined : (object ??= document.toObject(bsonOptions)),
+              options.noResponse
+                ? undefined
+                : 'moreToCome' in message && message.moreToCome
+                  ? { ok: 1 }
```

Comment:
The command monitoring spec tests expect `{ok: 1}` in the `event.reply` field for a unacknowledged write, [see here for the test details](https://github.com/mongodb/specifications/blob/39db05c6861f3ef6dba8f2cd00b3c7296f561bcd/source/command-logging-and-monitoring/tests/logging/unacknowledged-write.json#L112-L119).

Reply:
could noResponse also use ok:1? 

I know there's a test that expects undefined, but changing it to ok:1 doesn't break the TS API given its flexibility and communicates the same lack of information about the actual result of the command.

Reply:
I'd rather not add new functionality to `noResponse` if we're saying that it buggy and deprecated. What do you think?

Reply:
will do in NODE-6377